lib/ansible/modules/monitoring/zabbix/zabbix_group_info.py

```diff
@@ -40,6 +40,7 @@
             - Name of the hostgroup in Zabbix.
             - hostgroup is the unique identifier used and cannot be updated using this module.
         required: true
+        type: list
```

Comment:
```suggestion
        type: list
        elements: <type of elements>
```
probably ```str```, and so on, would be cool to see types of list elements determined for all params where possible

Reply:
Added `elements` where possible. I hope that `elements: dict` is valid as well

Reply:
It is valid.